internal/storage/db/storage.go

```diff
@@ -63,18 +64,27 @@ func (st *Storage) Ping(ctx context.Context) error {
 	return st.DB.PingContext(ctx)
 }
 
-func (st *Storage) Get(short string) (string, bool) {
-	var uri string
+func (st *Storage) Get(short string) (*storage.URIsItem, bool) {
```

Comment:
В индустрии чаще возвращают ошибку выше
```suggestion
func (st *Storage) Get(short string) (*storage.URIsItem, errors) {
```